src/plugin.js

```diff
@@ -258,13 +269,17 @@ const onPlayerReady = (player, emeError) => {
       try {
         handleMsNeedKeyEvent(event, getOptions(player), player.eme.sessions, player.tech_);
       } catch (error) {
-        emeError(error);
+        emeError(error, videojs.Error.MSKeyError);
       }
     });
-    player.tech_.on('mskeyerror', emeError);
+    const msKeyErrorCallback = (error) => {
+      emeError(error, videojs.Error.MSKeyError);
```

Comment:
Why do we have key system-specific errors?
I think we should use common interface for all of them

Reply:
discussed offline